recipes/cmiles/meta.yaml

```diff
@@ -0,0 +1,44 @@
+{% set name = "cmiles" %}
+{% set version = "0.1.5" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/openforcefield/cmiles/archive/v0.1.5.tar.gz
+  sha256: 866e1d56ae4f86160fcbeeb782f688a6d4b1f14c88ed23341a74a1b3f2f04bbc
+
+build:
+  noarch: python
+  number: 0
+  script: "{{ PYTHON }} -m pip install . -vv"
+
+requirements:
+  host:
+    - python
+    - pip
+  run:
+    - python
+    - rdkit
+
+test:
+  imports:
+    - cmiles
+
+about:
+  home: https://github.com/openforcefield/cmiles
+  license: MIT
+  license_family: MIT
+  license_file: LICENSE
+  summary: Generate canonical molecule identifiers for quantum chemistry
+  doc_url: https://cmiles.readthedocs.io/
+  dev_url: https://github.com/openforcefield/cmiles
+
+extra:
+  recipe-maintainers:
+    - mattwthompson
+    - j-wags
+    - dotsdl
+    - jaimergp
+    - jthorton
```

Comment:
we need confirmation from all of those maintainers listed here

Reply:
Confirmed!

Reply:
Confirmed!

Reply:
I agree to be a maintainer for this recipe. Thanks!

Reply:
Confirmed!